servers/lib/README.md

```diff
@@ -40,6 +40,37 @@ your Github username and the password is your Github
 In order for the npm to download the package, your personal access token
 needs to have _read:packages_ scope.
 
+## Use in Docker Environment
+
+### Adjust Configuration (Optional)
+
+The microservices require configuration,
+see the [Configure](#gear-configure) section for more info.
+
+The docker version of the microservices uses the configuration
```

Comment:
Changing the `config/.env.default` does not change the configuration of libms running inside docker container. In addition, the two most important environment variables, namely PORT and LOCAL_PATH are hardcoded into `compose.lib.yml`. What if we update `compose.lib.yml` as follows?

```yml
version: '3'
services:
  libms:
    image: intocps/libms:latest
    restart: unless-stopped
    volumes:
      - ${LOCAL_PATH}:/dtaas/libms/files
    ports:
      - "${PORT}:4001"
```

The variables get picked up from `.env` file. If user wants to specify a different env file, they can do so using

```bash

$docker compose \
  --env-file <filename> \
   -f compose.lib.yml up -d
```

If this makes sense, please update the READMEs accordingly. Thanks.

Reply:
@nichlaes please address this comment

Reply:
If using lib.docker.dev.yml it will

Reply:
Good Idea, I'll update the docker compose file

Reply:
@prasadtalasila This is not possible, due to how  docker works with environments.

So the environments in the container are set by `env-file`.
But the environments that docker compose can use are fetched by the .env or variables defined in terminal.